mucgpt-frontend/src/components/UnifiedHistory/UnifiedSidebarHistory.module.css

```
@@ -0,0 +1,193 @@
+    min-height: 30px;
+}
+
+.menuPopover {
+    overflow: hidden;
+}
+
+.menuList {
+    padding: 4px;
+}
+
+.menuList :global(.fui-MenuItem) {
```

Stylelint (17.9.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)